### ISSUE TITLE
Cannot add TV to Home app, but is working fine in Homebridge web UI

### ISSUE THREAD

**Issue by hylkefaber** [merdok/homebridge-webos-tv#305]
I am unable to add 2019 B9 TV to the iOS home app. However, the plugin works as expected using the homebridge-config-ui-x web-ui.

I know I have to manually add my TV to the Home app. When I do this, two instances of the TV show up: both the native HomeKit integration (LG webOS TV) and this plugin (LG TV) as defined in my config.json. However, selecting the plugin's TV results in an "unable to add accessory" message after trying for a while. 
I have tried resetting homebridge, deleting the persist and accessories folders, and then adding back every accessory manually, but the TV still doesn't work.
As the web-ui does work, I have no idea what I can try next. I think my config.json is correct, otherwise the web-ui should give issues, but it doesn't. Why does it work in web-ui but not on iOS? Any suggestions?

Running Homebridge v1.1.6 and homebridge-webos-tv v2.0.5, TV is 2019 B9 model.

**Comment by merdok** [comment]
Hmmm strange.
Did you try to disable the tv HomeKit integration, also do you have any errors in the homebridge debug log when adding the tv?

**Comment by hylkefaber** [comment]
I cannot seem to find how to disable the native LG integration. There's no errors on my homebridge log: [https://pastebin.com/ut1TP1Xq](https://pastebin.com/ut1TP1Xq)

**Comment by merdok** [comment]
From the log point of view everything looks ok with the plugin. Seems that you have some issues with your home app. Maybe some kind of cache issue? Try to clear the homebridge cache, also make sure that you are using the correct code when pairing, eventually try to rename the tv.

**Comment by hylkefaber** [comment]
Did all the things you mentioned already: changed TV name, cleared cache by removing the persist and accessories folder, changed username, but still no bueno, unfortunately

**Comment by merdok** [comment]
Then you have some issues with your homebridge installation or your Home in HomeKit. You need to investigate it and find the cause. 
It seems like your issue is not really related to this plugin.

**Comment by hylkefaber** [comment]
That's the thing, that's what I would think, but I have one other plugin for my thermostat, and everything works as expected there. Also, web-ui is working as it should, just not on the iPhone side

**Comment by merdok** [comment]
Yes, that is the point why I think that the issue is related to a local problem on your side, since the Webui is working. 
People usually try to delete the home from they HomeKit or reinstall homebridge not sure what other stuff I can recommend.

**Comment by hylkefaber** [comment]
Fixed it: I had to add a rule to `ufw` in order to allow Homebridge to let devices connect to this plugin. Not sure why, but I also had to specifically add ports 8581 and the HB assigned port via the `sudo ufw allow <port number>` command. For anyone who also has this problem: during startup of Homebridge, in /var/lib/homebridge/homebridge.log, this plugin announces the port that is used to communicate. For me, this was just before the homebridge connect code is shown in the log file. Allow this port in UFW to fix this problem.

**Comment by hylkefaber** [comment]
Fixed it: I had to add a rule to ufw in order to allow Homebridge to let devices connect to this plugin. Not sure why, but I also had to specifically add ports 8581 and the HB assigned port via the sudo ufw allow <port number> command. For anyone who also has this problem: during startup of Homebridge, in /var/lib/homebridge/homebridge.log, this plugin announces the port that is used to communicate. For me, this was just before the homebridge connect code is shown in the log file. Allow this port in UFW to fix this problem.
